db_scripts/mosip_esignet/ddl/add_thumbprint_public_key_repository.sql

```diff
@@ -0,0 +1,3 @@
+--adding new column thumbprint to public_key_registry
+ALTER TABLE public_key_registry
+ADD COLUMN thumbprint varchar;
```

Comment:
This upgrade db scripts are not as per the latest convention. Please follow the below repo branch as example 
https://github.com/mosip/audit-manager/tree/release-1.2.0.1/db_upgrade_scripts/

Reply:
done some privileges changes